libr/core/graph.c

```diff
@@ -2144,7 +2144,13 @@ R_API RANode *r_agraph_add_node (const RAGraph *g, const char *title,
 		sdb_array_add (g->db, "agraph.nodes", res->title, 0);
 		b = strdup (res->body);
 		len = strlen (b);
-		if (b[len - 1] == '\n') b[len - 1] = '\0';
+		if (len>0){
+			if (b[len - 1] == '\n') 
+				b[len - 1] = '\0';
+		}
+		else {
```

Comment:
Why the else? if len is not greater than 0 it's equal to zero and it means b already points to a '\0'.